sdk/rust/macro/src/lib.rs

```diff
@@ -6,17 +6,28 @@ use quote::quote;
 pub fn http_component(_attr: TokenStream, item: TokenStream) -> TokenStream {
     let func = syn::parse_macro_input!(item as syn::ItemFn);
     let func_name = &func.sig.ident;
+    const HTTP_COMPONENT_WIT_PATH: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/wit");
 
     quote!(
         #func
 
         mod __spin_http {
+            #![allow(missing_docs)]
+            ::spin_sdk::wit_bindgen::generate!({
+                world: "exports",
+                path: #HTTP_COMPONENT_WIT_PATH,
```

Comment:
I don't think this will work in general, given that the wit directory may not be present on the machine building the application code that is invoking the macro, right?

Reply:
TIL this should work since the wit directory will be packaged as part of the crate.